rfc/rfc011-verifiable-credential.md

```diff
@@ -48,7 +48,43 @@ All VCs MUST use DIDs as specified in [RFC004](rfc004-distributed-document-forma
 
 The following proof types must be supported:
 
-* JsonWebSignature2020 (https://w3c-ccg.github.io/ld-proofs/ and https://tools.ietf.org/html/rfc7797)
+#### 3.1.1 JsonWebSignature2020
+
+resources:
+
+* https://w3c-ccg.github.io/ld-proofs/ 
+* https://tools.ietf.org/html/rfc7797
+* https://w3c-ccg.github.io/lds-jws2020
+
+The *JsonWebSignature2020* refers to https://w3id.org/rdf#URDNA2015 canonicalization but it's unclear how this is to be implemented for json, therefore it's skipped for now.
+It's quite hard to decipher those specifications and they seem to be very drafty as well. The algorithm used to create the signature is as follows:
+
+- compose a `proof` json object:
+
+```json
+{
+  "type": "JsonWebSignature2020",
+  "proofPurpose": "assertionMethod",
+  "verificationMethod": "<<kid>>",
+  "created": "<<RFC3339>>"
+}
+```
+where `<<kid>>` is replaced with an assertionMethod ID from the DID Document and `<<RFC3339>>` is replaced with a RFC3339 compliant time string.
+
+- take the sha256 of the `proof` and concatenate with the sha256 of the input verifiable credential (without the proof)
```

Comment:
Can you write a bit of pseudo code here? Something like 

`b64_encode(sign(..), )` (etc)